ergast_py/type_constructor.py

```diff
@@ -199,6 +200,18 @@ def construct_races(self, races: dict) -> list[Race]:
         """
         return [self.construct_race(race) for race in races]
 
+    def construct_time(self, time: dict) -> Time:
+        """
+        Construct a Time from a JSON dictionary
+        """
+        if time != {}:
+            return Time(
+                millis=Helpers().construct_lap_time_millis(millis=time),
```

Comment:
`Helpers().construct_lap_time_millis()` can throw an exception. We should try to protect this from throwing the error out (see [construct_result](https://github.com/Samuel-Roach/ergast-py/blob/develop/ergast_py/type_constructor.py#L247-L251) for an example of this)